backend/src/main/java/com/my/shope/backend/product/ProductController.java

```diff
@@ -86,5 +85,9 @@ public void orderAll(){
     }
 
 
+    @GetMapping("/shopping-cart-size")
+    public int getShoppingCartSiz(){
```

Comment:
nur ein kleiner Typo in `Siz` statt `Size`

Reply:
danke :)